### PR TITLE
Fix abnormal cpu usage while syncing

### PR DESCRIPTION
**Pull request by sgaragagghu**
Fix abnormal cpu usage while syncing (downloader side)

### CODE REVIEW

**Comment by sgaragagghu** [comment]
it depend on #247

**Comment by sgaragagghu** [comment]
fixed in #259

File: client/sync.c [hunk 1]
```diff
@@ -105,47 +110,54 @@ int xdag_sync_pop_block(struct xdag_block *b)
...
 		}
 	}
 
-	pthread_mutex_unlock(&g_sync_hash_mutex);
+//	pthread_mutex_unlock(&g_sync_hash_mutex);
 
 	return 0;
 }
 
+int xdag_sync_pop_block(struct xdag_block *b){
```
Comment (jonano614) [inline client/sync.c]: Put braces on a new line for functions.

**Comment by jonano614** [comment]
Am I right the changes are only in the sync.c here? Lets wait for PR #247

**Pull request closed without merging**